Cod3rsGrowth.Infra/Repositorio/JogoRepositorio.cs

```diff
@@ -1,11 +1,21 @@
-﻿using Cod3rsGrowth.Dominio.Entidades;
+﻿using Cod3rsGrowth.Dominio;
+using Cod3rsGrowth.Dominio.Entidades;
 using Cod3rsGrowth.Infra.Interfaces;
+using LinqToDB;
 using System.Collections.Generic;
+using System.Linq;
 
 namespace Cod3rsGrowth.Infra.Repositorio
 {
     public class JogoRepositorio : IJogoRepositorio
     {
+        DbCod3rsGrowth bancoDeDados;
+
+        public JogoRepositorio(DbCod3rsGrowth bancoDeDados)
+        {
+            this.bancoDeDados = bancoDeDados;
```

Comment:
essa variável deve ser privada e somente de leitura, ao colocar o nome da variavel no padrão do C#, não será mais necessário a cláusula this